theano/tensor/nnet/tests/test_corr.py

```diff
@@ -416,6 +417,45 @@ def test_non_contiguous(self):
         self.validate((3, 2, 7, 5), (5, 2, 2, 3), 2, non_contiguous=True)
 
 
+class TestGroupCorr2d(Grouped_conv_noOptim):
+    if theano.config.mode == "FAST_COMPILE":
+        mode = theano.compile.get_mode("FAST_RUN")
+    else:
+        mode = None
+    conv2d = staticmethod(corr.CorrMM)
+    conv2d_gradw = staticmethod(corr.CorrMM_gradWeights)
+    conv2d_gradi = staticmethod(corr.CorrMM_gradInputs)
+    flip_filter = True
+
+    def test_graph(self):
```

Comment:
This test was implemented in accordance with this comment https://github.com/Theano/Theano/pull/5991#discussion_r119887385 in the second paragraph.